rest/snippets/Media/MediaPlaySession.yml

```diff
@@ -15,14 +15,6 @@ MediaPlaySession:
     ip4_address:
       type: string
       description: The ip address of the client that created this session
-    channel_id:
-      type: string
-      description: >-
-        This key will only be developed as part of the out-of-home restrictions and will not be implemented immediately.
```

Comment:
It seems [Andy added](https://github.com/booxmedialtd/api-documentation/commit/f9ba0cdb8725013ade10f8aef9265f71964c1b59#diff-57ea38f653de3dbadb0c6602d456e6a9) this attribute for our-of-home epic. 
@evgenii-aminocom do we want to keep it as is?

Reply:
@amino-garricklam now I see what you mean.
But no, out-of-home epic was redesigned since then and works via stream APIs, no connection to Beaver.